rivus/utils/prerun.py

```diff
@@ -0,0 +1,45 @@
+"""
+    Collection of small rivus related util functions
+    
+    In use to avoid multiple solutions of the same
+    problem.
+"""
+from multiprocessing import cpu_count
+
+def setup_solver(optim, logfile='solver.log', guroTimeLimit=12000, guroMIPFocus= 2, guroMIPGap=.001, guroThreads=None):
```

Comment:
Ok. After the first touches of rivus I turned off my autoPEP8 because there was too much going on and I did not want to pep8-ify as a robot.
Diffs: 
+ trailing whitespace (solved in one of my first PRs),
+ length: I like 80-ish line length, but I admit to have more often 90<, I will work on it.
+ on-line comment styles
+ and [i,j,k] -> [i, j, k] () issues [pep8diff_rivusmain.diff.txt](https://github.com/tum-ens/rivus/files/1119407/pep8diff_rivusmain.diff.txt)

But I can gladly turn pep8 back on and revise my contribution.
I will also add some spellchecker to decimate typos in the future ;)

Thanx for the code reviews, I like this in co-operation very much.